app/components/ListPlusCell.tsx

```diff
@@ -18,14 +19,20 @@ type ListPlusCellProps = {
 }
 
 /**
- * Gives a count with a tooltip that expands to show details when the user hovers over it
+ * Gives a count with a tooltip that expands to show details when the user hovers over it.
+ * The ReactNode children are split into two groups: the first `numInCell` are shown in the cell,
+ * and the rest are shown in the tooltip. If the number of children is less than or equal to
+ * `numInCell`, no tooltip (or `+N` target) is shown.
  */
 export const ListPlusCell = ({
   tooltipTitle,
   children,
   numInCell = 1,
 }: ListPlusCellProps) => {
   const array = React.Children.toArray(children)
+  if (array.length === 0) {
+    return <EmptyCell />
+  }
   const inCell = array.slice(0, numInCell)
   const rest = array.slice(numInCell)
   const content = (
```

Comment:
I think we should add a class to the `tooltipTitle` div to style the text. We never noticed this before because the items in the list were all badges, so they were distinct from the title text already. Now that they're text, I think the title should be white or bold or both.

![image](https://github.com/user-attachments/assets/e3f80cbd-c2c2-430c-a1f0-cb2414d4f624)

Reply:
Updated to `text-sans-semi-md text-default`

<img width="279" alt="Screenshot 2024-09-17 at 9 46 46 AM" src="https://github.com/user-attachments/assets/3803131a-4ea0-49dc-9ee7-c0a34d7a0269">